Extension/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # C/C++ for Visual Studio Code Change Log
 
-## Version 0.17.7: July 16, 2018
+## Version 0.17.7-insiders: July 12, 2018
```

Comment:
don't change the changelog. it's a source of merge conflicts later.

Reply:
Okay.